Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -92,51 +92,56 @@ public async Task ReadUserMessages(MessagesCommand command, Contexts contexts)
         }
 
         [AdminCommand]
-        public async Task SetRoleAsSafe(SetRoleCommand command, Contexts contexts)
+        public Task SetRoleAsSafe(SetRoleCommand command, Contexts contexts)
         {
-            if (!command.Safe && !command.Unsafe)
+            if (command.Safe == command.Unsafe)
             {
+                if (command.Safe)
```

Comment:
nie wiem czy to jest już zabezpieczone
ale przy generowaniu przykładowych komend, nie powinno generować przykładów gdzie są 2 boole jednocześnie

Reply:
no generowanie jest już tak zrobione że tylko jeden bool, ale co to ma do tego?

Reply:
do tego nic, po prostu mi sie przypomniało jak spojrzałem na to